frontend/src/metabase/dashboard/actions/data-fetching-typed.ts

```diff
@@ -124,10 +123,6 @@ export const fetchDashboard = createAsyncThunk(
 
       fetchDashboardCancellation = null;
 
-      if (dashboardType === "normal" || dashboardType === "transient") {
```

Comment:
We now don't block `query` endpoints until metadata is loaded. The same check is moved to `fetchDashboardCardMetadata`